tests/helpers/aptible-helpers.js

```diff
@@ -97,6 +97,15 @@ Ember.Test.registerHelper('stubApp', function(app, appData){
   });
 });
 
+Ember.Test.registerHelper('stubDatabase', function(app, databaseData){
+  var id = databaseData.id;
+  if (!id) { throw new Error('cannot stub database without id'); }
+
+  stubRequest('get', '/databases/' + id, function(request){
```

Comment:
this could be `stubRequest('get', '/databases/:db_id/', function(request){`, and then the response could dynamically fill in the id from `request.params.db_id`. But stubbing *any* db get call to return `databaseData` isn't the intent of this helper, so I think it makes sense to hard-code to the given id.